### DISCUSSION TITLE
Actividad 1 - GRUPO 3

### DISCUSSION THREAD

**Discussion by renatacatelli** [frlp-utn-ingsoft/catedra#92] [Show and tell]
https://github.com/BogadoLucasDemian/vetsoft

**Comment by ivanduva** [comment]
Buenas!
La actividad está bien resuelta, queda aprobada.
Les dejamos algunos comentarios para que tengan en cuenta en la próxima:

- Hay algunos tests que quedaron rotos. Como la app funciona y además esta actividad no se trataba de tests, no hay problema, pero a partir de la próxima actividad no puede quedar ningún test fallando.
- Se puede mejorar la estandarización de los mensajes de commit. Vimos que lo intentaron hacer para el caso de los fixes y lo valoramos, pero quedan un poco difíciles de seguir los números porque no están basados en ningún issue o ticket. Lo mismo para el caso de los branches.
- Tengan cuidado con las migraciones, vimos que se olvidaron de subirlas y algunas quedaron en los últimos commits. El problema con eso es que en los commits anteriores la app puede no funcionar.
- Si bien no es algo generalizado, hay algunos commits no atómicos como este: https://github.com/BogadoLucasDemian/vetsoft/commit/a10a88c88702f17c7b9307ca37292a84fe8cc5f1

Buen trabajo!
Saludos
